### PR TITLE
8341072: Open source several AWT Canvas and Rectangle related tests

### PR DESCRIPTION
**Pull request by TejeshR13**
Open source these AWT Canvas and Rectangle tests:
java/awt/Canvas/MultiDitherTest/MultiDitherTest.java
java/awt/Canvas/MultiGraphicsTest/MultiGraphicsTest.java
java/awt/Canvas/NoEventsLeakTest/NoEventsLeakTest.java
java/awt/Rectangle/IntersectionTest/IntersectionTest.java

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8341072](https://bugs.openjdk.org/browse/JDK-8341072): Open source several AWT Canvas and Rectangle related tests (**Bug** - P4)

### Reviewers
 * [Phil Race](https://openjdk.org/census#prr) (@prrace - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/21464/head:pull/21464` \
`$ git checkout pull/21464`

Update a local copy of the PR: \
`$ git checkout pull/21464` \
`$ git pull https://git.openjdk.org/jdk.git pull/21464/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 21464`

View PR using the GUI difftool: \
`$ git pr show -t 21464`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/21464.diff">https://git.openjdk.org/jdk/pull/21464.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/21464#issuecomment-2406700781)

### CODE REVIEW

File: test/jdk/java/awt/Canvas/MultiGraphicsTest.java [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+                Depending on the GraphicsConfiguration, images may be
+                displayed in color or in grayscale and/or displayed at a
+                lower bitdepth.
+                The number of GraphicsConfigurations will be printed below
+                as the test is starting up.
+                On Solaris, making some Windows active will cause some of
+                the other Windows to display strangely. This is normal.
+                Ensure that there are as many Windows created as there are
+                available GraphicsConfigurations.
+                Examine each Window to ensure it displays Jim.
```
Comment (prrace) [inline test/jdk/java/awt/Canvas/MultiGraphicsTest.java:68]: Jim ? You mean Duke.

Also remove mention of Solaris, it hasn't been a supported platform for several years

File: test/jdk/java/awt/Canvas/MultiDitherTest.java [hunk 2]
```diff
@@ -0,0 +1,426 @@
...
+        String INSTRUCTIONS = """
+                This test displays several dithered Canvases similar to the
+                DitherTest demo applet, one for each available GraphicsConfiguration.
+                Depending on the GraphicsConfiguration, the dithering may be in
+                color or in grayscale and/or display at a lower bitdepth.
+                The number of GraphicsConfigurations will be printed in the
+                TextArea below as the test is starting up..
+                On Solaris, making some Windows active will cause some of the
+                other Windows to display strangely. This is normal.
+                Ensure that there are as many Frames created as there are
```
Comment (prrace) [inline test/jdk/java/awt/Canvas/MultiDitherTest.java:84]: remove the sentence about Solaris, it hasn't been a supported platform for several years

File: test/jdk/java/awt/Canvas/MultiDitherTest.java [hunk 3]
```diff
@@ -0,0 +1,426 @@
...
+    static LayoutManager dcLayout = new FlowLayout(FlowLayout.CENTER, 10, 5);
+    Thread runner;
+    DitherControls XControls;
+    DitherControls YControls;
+    DitherCanvas canvas;
+
+    public static void main(String[] args) throws Exception {
+        String INSTRUCTIONS = """
+                This test displays several dithered Canvases similar to the
+                DitherTest demo applet, one for each available GraphicsConfiguration.
```
Comment (prrace) [inline test/jdk/java/awt/Canvas/MultiDitherTest.java:77]: remove the phrase "similar to the DitherTest demo applet" that applet was retired many years ago.

File: test/jdk/java/awt/Rectangle/IntersectionTest.java [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+ * questions.
+ */
+
+/*
+ * @test
+ * @bug 4147957
+ * @key headful
+ * @summary Test to verify setClip with invalid rect changes rect to valid
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
```
Comment (prrace) [inline test/jdk/java/awt/Rectangle/IntersectionTest.java:30]: This isn't a manual test, it does not need these two lines

Comment (TejeshR13) [reply test/jdk/java/awt/Rectangle/IntersectionTest.java]: Updated.

File: test/jdk/java/awt/Canvas/MultiDitherTest.java (hunk 3 above)
Comment (TejeshR13) [reply test/jdk/java/awt/Canvas/MultiDitherTest.java]: Removed the phrase.

File: test/jdk/java/awt/Canvas/MultiDitherTest.java (hunk 2 above)
Comment (TejeshR13) [reply test/jdk/java/awt/Canvas/MultiDitherTest.java:80]: Removed.

File: test/jdk/java/awt/Canvas/MultiGraphicsTest.java (hunk 1 above)
Comment (TejeshR13) [reply test/jdk/java/awt/Canvas/MultiGraphicsTest.java]: Updated.

**Review by prrace — APPROVED** [review]

**Pull request closed without merging**